.pylintrc

```diff
@@ -60,85 +60,14 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use "--disable=all --enable=classes
 # --disable=W".
-disable=print-statement,
```

Comment:
A bunch of these rules got deprecated from pylint. I think they had to do with the Python 2 -> 3 transition